jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -124,10 +124,7 @@ function BindUsernameToJCSystem {
         }
     }
     Process {
-        # Get UserID from JumpCloud Console
-        Write-ToLog -Message:("Searching for user: $JumpCloudUserName")
-        $ret, $id = Test-JumpCloudUsername -JumpCloudApiKey $JcApiKey -JumpCloudOrgID $JcOrgId -Username $JumpCloudUserName
-        if ($ret -And $id) {
+        if ($JumpCloudId) {
```

Comment:
Do we need this IF statement here? I doubt it since our BindUsernameToJCSystem has the parameter listed as mandatory.